### ISSUE TITLE
sift mixes up projections and areadefinitions

### ISSUE THREAD

**Issue by gerritholl** [ssec/sift#393]
Under the "Projection:" dropdown menu, the listed labels refer not to projections but to AreaDefinitions, although some of the information contained in an `AreaDefinition` is not really used.  The projection is identical between MTG FCI FDSS 1km and MTG FCI FDSS 2 km.  The extent isn't really used; for example, if I load `EUROL`, I can zoom out as far as I want (in the extreme, this leads to artefacts).

**Comment by djhoese** [comment]
I think a similar point was made by me either here in this repository or on the EUM gitlab during their primary development phase where I pointed out that it doesn't make sense (to me at least) to let users choose the area definition with a resolution associated with it in the Map Canvas projection as the GPU-side of SIFT doesn't actually care about resolution. It only cares about projection. Otherwise, everything is just a pixel with a coordinate next to other pixels with coordinates.

**Comment by ameraner** [comment]
Indeed, that part is a bit confusing/misleading at the moment. The reason why atm we list multiple AreaDefinitions with the same projection but different resolutions is the fact that the resolution (array shape) needs to be defined for when the user asks for explicit resampling in the Open File Wizard (using one of the available  resamplers). A better design would be to keep only projections in the main window, and move the resolution selection into the Open File Wizard..
